targets/TARGET_NUVOTON/TARGET_M451/TARGET_NUMAKER_PFM_M453/device.h

```diff
@@ -38,7 +38,7 @@
 #define DEVICE_SPI_ASYNCH       1
 #define DEVICE_SPISLAVE         1
 
-#define DEVICE_CAN              0
+//#define DEVICE_CAN              1
```

Comment:
That's correct but we should not leave dead  code in the sources. Remove it completely?